static/ca_bydate.js

```diff
@@ -283,12 +290,28 @@ function parsingDone(results, file) {
       drawChart();
     }
   });
+
+  redoFilter();
+
+  _.each([countySel, locationTypeSel], function(sel) {
+    sel.addEventListener('change', function() {
+      county = countySel.value ? countySel.value : ALL
+      venue = locationTypeSel.value ? locationTypeSel.value : ALL;
+      window.location = "/bydatesel/" + county + "/" + venue;
```

Comment:
Use `encodeURIComponent()` for county and venue

Reply:
I was doing this, but noticed the browser is automatically encoding these, so took it out.  Is this a browser compatibility issue?